cellxgene_schema_cli/scripts/ontology_processing.py

```diff
@@ -0,0 +1,162 @@
+import yaml
+import urllib.request
+import os
+import gzip
+import json
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../cellxgene_schema"))
+import env
+
+
+def _download_owls(
+    owl_info_yml: str = env.OWL_INFO_YAML, output_dir: str = env.ONTOLOGY_DIR
+):
+
+    """
+    Downloads the ontology owl files specified in 'owl_info_yml' into 'output_dir'
+
+    :param str owl_info_yml: path to yaml file wit OWL information
+    :param str output_dir: path to writable directory where owl files will be downloaded to
+
+    :rtype None
+    """
+
+    with open(owl_info_yml, "r") as owl_info_handle:
+        owl_info = yaml.safe_load(owl_info_handle)
+
+    for ontology, info in owl_info.items():
+
+        print(f"Downloading {ontology}")
+
+        # Get owl info
+        latest_version = owl_info[ontology]["latest"]
+        url = owl_info[ontology]["urls"][latest_version]
+
+        # Format of owl (handles cases where they are compressed)
+        download_format = url.split(".")[-1]
+
+        output_file = os.path.join(output_dir, ontology + ".owl")
+        if download_format == "gz":
+            urllib.request.urlretrieve(url, output_file + ".gz")
+            _decompress(output_file + ".gz", output_file)
+            os.remove(output_file + ".gz")
+        else:
+            urllib.request.urlretrieve(url, output_file)
+
+
+def _decompress(infile: str, tofile: str):
+    """
+    Decompresses a gziped file
+
+    :param str infile: path gziped file
+    :param str tofile: path to output decompressed file
+
+    :rtype None
+    """
+    with open(infile, "rb") as inf, open(tofile, "w", encoding="utf8") as tof:
+        decom_str = gzip.decompress(inf.read()).decode("utf-8")
+        tof.write(decom_str)
+
+
+def _parse_owls(
+    working_dir: str = env.ONTOLOGY_DIR,
+    owl_info_yml: str = env.OWL_INFO_YAML,
+    output_json_file: str = env.PARSED_ONTOLOGIES_FILE,
+):
+
+    """
+    Parser all owl files in working_dir. Extracts information from all classes in the owl file.
+    The extracted information is written into a gzipped a json file with the following structure:
+    {
+        "ontology_name":
+            {
+            "term_id": {
+                "label": "..."
+                "ancestors": [
+                    "ancestor1_term_id_1",
+                    "ancestor2_term_id_2"
+                    ]
+                }
+            }
+
+            "term_id2": {
+                ...
+            }
+
+            ...
+            }
+    }
+
+    :param str working_dir: path to folder with owl files
+    :param str owl_info_yml: path to writable directory where owl files will be downloaded to
+    :param str owl_info_yml: path to yaml file wit owl information
+    :param str output_json_file: path to output jsaon file
+
+    :rtype None
+    """
+
+    with open(owl_info_yml, "r") as owl_info_handle:
+        owl_info = yaml.safe_load(owl_info_handle)
+
+    owl_files = [
```

Comment:
Can you refactor this for readability/clarity. Ideally avoid using single character variable names